src/core/dev_api/openvino/runtime/shared_buffer.hpp

```diff
@@ -36,6 +36,12 @@ class SharedStreamBuffer : public std::streambuf {
     explicit SharedStreamBuffer(const void* data, size_t size)
         : SharedStreamBuffer(reinterpret_cast<const char*>(data), size) {}
 
+    const char* view(std::streamsize count) const noexcept {
+        const auto real_count = std::min<std::streamsize>(m_size - m_offset, count);
+        if (real_count == count) // only return pointer when having enough size
```

Comment:
if (real_count == count)  // only return pointer when having enough size
```suggestion
        if (real_count == count)  // only return pointer when having enough size
```

Please update code style for failed test